src/Norm/base/ALString.h

```diff
@@ -307,7 +307,7 @@ inline ALString::ALString()
 {
 	Init();
 }
-#if not defined __UNIX__ or defined __C11__
+#if defined __C11__
```

Comment:
is `__C11__` defined when C++14 is used ?

Reply:
You're Hawkeye ! This a bug  `__C11__`  was defined in the CMakeLists that's no longer the case. I will change it with `__cplusplus`

Reply:
Ok !